### DISCUSSION TITLE
Hide spam log on runtime

### DISCUSSION THREAD

**Discussion by alecfilios** [gstavrinos/ai_msc_robotic_systems_assignments#11] [Q&A] [answered]
Is there a way to hide such messages during runtime?

`[rviz2-15] [INFO] [1686230714.266432166] [rviz]: Message Filter dropping message: frame 'base_laser_link' at time 60,792 for reason 'discarding message because the queue is full'
[component_container_isolated-16] [INFO] [1686230714.311177462] [amcl]: Message Filter dropping message: frame 'base_laser_link' at time 60.892 for reason 'the timestamp on the message is earlier than all the data in the transform cache'
`

**Comment by gstavrinos** [comment, marked as answer]
Generally, you would have to use something like this `output={"both": "log"}` in your launch files, for the node(s) that you do not want to log on screen (but rather on a logfile). Since the launch file used in the assignment includes other launch files, you might have to recursively edit files in order to achieve what you desire.
